checkSbitMappingAndRate.py

```diff
@@ -1,4 +1,77 @@
 #!/bin/env python
+r"""
+SBit Map Rate
+=============
+
+Investigate the sbit mapping and rate measurement in OH \& CTP7 FPGA. What does this mean? Who knows?
```

Comment:
Let's at least have the descriptions minimally correct, if you want a short executive summary, you can ask.
Otherwise, can add a 
```
FIXME Add detailed description
```

Reply:
Thanks! Should be updated now